### PR TITLE
Nm 204

### PR DESCRIPTION
**Pull request by SilviaKorcekova**

### CODE REVIEW

File: services/comments.service.js [hunk 1]
```diff
@@ -42,6 +42,165 @@ function getCommentsSum(comments, userName, startDate, endDate) {
...
+    });
+
+    return Promise.all(repositories.map(repositoryName => {
+        return githubService.getPullRequestsForRepository(repositoryName, pullRequestState);
+    }))
+        .then(pullRequestsPerRepository => {
+            return pullRequestsPerRepository.flatMap(pull => {
+                let pullRequestsData = pull.data;
+                //We count only with already merged pull requests
+                pullRequestsData = pullRequestsData.filter(pullRequest => pullRequest.merged_at != null);
```
Comment (tjalc) [inline services/comments.service.js:131]: we don't care when PR was merged?
In my task NM-198 I did check whether pullRequest.merged_at was between min_startDate and max_endDate from dateIntervals.
But not sure whether it is ok....

File: services/comments.service.js [hunk 2]
```diff
@@ -42,6 +42,165 @@ function getCommentsSum(comments, userName, startDate, endDate) {
...
+        }
+
+        return result;
+    });
+
+
+}
+
+function getCommentsCountForTeam(repositories, teamMembers, dateIntervals) {
+    let pullRequestState = "all";
```
Comment (tjalc) [inline services/comments.service.js:93]: why we are searching for 'all' PRs?
why not only CLOSED?

File: services/comments.service.js [hunk 3]
```diff
@@ -42,6 +42,165 @@ function getCommentsSum(comments, userName, startDate, endDate) {
...
+            teamComments: utils.calculateTrendValuesForDateIntervals(data.teamData, 'numberOfComments', undefined, 1)
+        }
+
+        return result;
+    });
+
+
+}
+
+function getCommentsCountForTeam(repositories, teamMembers, dateIntervals) {
```
Comment (tjalc) [inline services/comments.service.js:92]: maybe rename to getCommentsCountForREpositoriesAndTeamMembers

File: services/comments.service.js [hunk 4]
```diff
@@ -42,6 +42,165 @@ function getCommentsSum(comments, userName, startDate, endDate) {
     .then(commentsCount => JSON.parse(`{"userName":"${userName}", "commentsCount":${commentsCount}, "startDate":"${startDate}", "endDate":"${endDate}"}`));
 }
 
+function getCommentsCount(teamName, dateIntervals) {
```
Comment (tjalc) [inline services/comments.service.js:45]: I would rename this to getCommentsCountForTeam

File: services/comments.service.js (hunk 1 above)
Comment (SilviaKorcekova) [reply services/comments.service.js:131]: I count statistics for pull requests directly, so I filter only merged, that means unfilter closed and still opened pull requests. Then I am checking created_at in dateIntervals no merged_at. We can unite our approaches. This needs discussion.

**Pull request merged**